_includes/disqus_comments.html

```diff
@@ -0,0 +1,19 @@
+{% if page.comments != false %}
```

Comment:
Where does `page.comments` come from? Shouldn't this be checking if there is a disqus shortname?

Reply:
I'll fix all this and let you know, thanks for the feeback!

Reply:
I finally decided to let my friend finish the Disqus comments, I removed them from the forked repo so that you can merge. He'll open a new pr for disqus.